app/src/main/java/com/kickstarter/viewmodels/EmailVerificationInterstitialFragmentViewModel.kt

```diff
@@ -4,9 +4,40 @@ import androidx.annotation.NonNull
 import com.kickstarter.libs.Environment
 import com.kickstarter.libs.FragmentViewModel
 import com.kickstarter.ui.fragments.EmailVerificationInterstitialFragment
+import rx.Observable
+import rx.subjects.BehaviorSubject
 
 class EmailVerificationInterstitialFragmentViewModel {
-    interface Inputs { }
-    interface Outputs { }
-    class ViewModel(@NonNull val environment: Environment) : FragmentViewModel<EmailVerificationInterstitialFragment>(environment), Outputs, Inputs { }
+    interface Inputs {
+        /** Invoked when the open inbox button is pressed */
+        fun openInboxButtonPressed()
+    }
+
+    interface Outputs {
+        fun startEmailActivity(): Observable<Void>
+    }
+
+    class ViewModel(@NonNull val environment: Environment) : FragmentViewModel<EmailVerificationInterstitialFragment>(environment), Outputs, Inputs {
+        val inputs = this
+        val outputs = this
+
+        private val openInboxButtonPressed = BehaviorSubject.create<Void>()
+
+        private val startEmailActivity = BehaviorSubject.create<Void>()
+
+
+        init {
+            openInboxButtonPressed
```

Comment:
add in here `this.openInboxButtonPressed` we do explicitly call `this.` for the global variables

Reply:
got it 👍